Kiss.Bff/Config/AuthenticationSetup.cs

```diff
@@ -55,36 +58,50 @@ public static IServiceCollection AddKissAuth(this IServiceCollection services, s
                 //options.Events.OnSigningOut = (e) => e.HttpContext.RevokeRefreshTokenAsync();
                 options.Events.OnRedirectToAccessDenied = HandleLoggedOut;
                 options.Events.OnRedirectToLogin = HandleLoggedOut;
-            })
-            .AddOpenIdConnect(ChallengeSchemeName, options =>
-            {
-                options.NonceCookie.HttpOnly = true;
-                options.NonceCookie.IsEssential = true;
-                options.NonceCookie.SameSite = SameSiteMode.None;
-                options.NonceCookie.SecurePolicy = CookieSecurePolicy.Always;
-                options.CorrelationCookie.HttpOnly = true;
-                options.CorrelationCookie.IsEssential = true;
-                options.CorrelationCookie.SameSite = SameSiteMode.None;
-                options.CorrelationCookie.SecurePolicy = CookieSecurePolicy.Always;
-
-                options.Authority = authority;
-                options.ClientId = clientId;
-                options.ClientSecret = clientSecret;
-                options.SignedOutRedirectUri = SignOutCallback;
-                options.ResponseType = OidcConstants.ResponseTypes.Code;
-                options.UsePkce = true;
-                options.GetClaimsFromUserInfoEndpoint = true;
-
-                options.Scope.Clear();
-                options.Scope.Add(OidcConstants.StandardScopes.OpenId);
-                options.Scope.Add(OidcConstants.StandardScopes.Profile);
-                options.Scope.Add(OidcConstants.StandardScopes.OfflineAccess);
-                options.SaveTokens = true;
-
-                options.Events.OnRemoteFailure = RedirectToRoot;
-                options.Events.OnSignedOutCallbackRedirect = RedirectToRoot;
             });
 
+            if (!string.IsNullOrWhiteSpace(authority))
```

Comment:
wat is de relatie met de story?

Reply:
Ook hiervoor geldt dat de applicatie klapt als er geen waarde is ingevuld voor authority. Dan krijg ik de integratietest niet werkend.

Reply:
worden issues door ontbrekende/ongeldige release vars in productie niet moeilijker op te sporen hierdoor? misschien ter compensatie iets loggen?

Reply:
goed plan, voeg ik toe